src/OpenTelemetry/CHANGELOG.md

```diff
@@ -77,6 +77,16 @@
   Specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/configuration/sdk-environment-variables.md#exemplar).
   ([#5412](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5412))
 
+* `TracerProvider`s can now have a sampler configured via the
```

Comment:
Please update the doc here too to reflect new capability : https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/trace/customizing-the-sdk/README.md#samplers